source/jormungandr/jormungandr/interfaces/v1/serializer/api.py

```diff
@@ -265,3 +269,31 @@ class TrafficReportsSerializer(PTReferentialSerializer):
 
 class CalendarsSerializer(PTReferentialSerializer):
     calendars = pt.CalendarSerializer(many=True, display_none=True)
+
+
+class StatusSerializer(serpy.DictSerializer):
+    status = status.StatusSerializer()
+
+
+class GeoStatusSerializer(serpy.DictSerializer):
+    geo_status = geo_status.GeoStatusSerializer()
+
+
+class GraphicalIsrochoneSerializer(serpy.Serializer):
+    isochrones = graphical_isochron.GraphicalIsrochoneSerializer(attr='graphical_isochrones', many=True)
+    error = ErrorSerializer(display_none=False, attr='error')
+    feed_publishers = FeedPublisherSerializer(many=True, display_none=True)
+    links = MethodField(schema_type=LinkSchema(many=True), display_none=True)
+    warnings = base.BetaEndpointsSerializer()
+
+    def get_links(self, obj):
+        return _get_links_impl(obj)
```

Comment:
Maybe you can do a JourneyCommonSerilalizer that manages almost all that.

Reply:
done